web.config

```diff
@@ -116,32 +229,18 @@
           See https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config for a full list of options
         -->
         <!-- <iisnode watchedFiles="web.config;*.js"/> -->
-        <staticContent>
-            <remove fileExtension=".css"/>
-            <mimeMap fileExtension=".css" mimeType="text/css; charset=utf-8"/>
-            <remove fileExtension=".html" />
-            <mimeMap fileExtension=".html" mimeType="text/html; charset=utf-8" />
-            <remove fileExtension=".js"/>
-            <mimeMap fileExtension=".js" mimeType="text/javascript; charset=utf-8"/>
-            <remove fileExtension=".json"/>
-            <mimeMap fileExtension=".json" mimeType="application/json; charset=utf-8"/>
-            <remove fileExtension=".svg"/>
-            <mimeMap fileExtension=".svg" mimeType="image/svg+xml; charset=utf-8"/>
-            <remove fileExtension=".svgz"/>
-            <mimeMap fileExtension=".svgz" mimeType="image/svg+xml"/>
-            <remove fileExtension=".ttf"/>
-            <mimeMap fileExtension=".ttf" mimeType="font/ttf"/>
-            <remove fileExtension=".woff"/>
-            <mimeMap fileExtension=".woff" mimeType="font/woff"/>
-            <remove fileExtension=".woff2"/>
-            <mimeMap fileExtension=".woff2" mimeType="font/woff2"/>
-            <remove fileExtension=".xml"/>
-            <mimeMap fileExtension=".xml" mimeType="application/xml; charset=utf-8"/>
-            <remove fileExtension=".webmanifest"/>
-            <mimeMap fileExtension="webmanifest" mimeType="application/manifest+json; charset=utf-8"/>
-        </staticContent>
+
     </system.webServer>
     <system.web>
         <customErrors mode="RemoteOnly" defaultRedirect="/dist/404/" redirectMode="ResponseRewrite"/>
     </system.web>
+    <!-- All our static assets should be under `dist/static`, we set a long cache and the `immutable` directive, overriding
+         the `no-cache` we set up earlier -->
+    <location path="dist/static">
+        <system.webServer>
+            <staticContent>
+                <clientCache cacheControlMode="UseMaxAge" cacheControlMaxAge="365.00:00:00" cacheControlCustom="immutable" />
```

Comment:
How about leap years when there are 366 days? Can this directive accept 1 year instead of certain number of days?

Reply:
Not AFAIK. Also I don’t see why some years we should cache it 365 and others 366, we just need an arbitrary length and 365 is kind of the standard and what our rule tests for.

Reply:
>  and what our rule tests for.

Ah ok.